configs/local_example.yaml

```diff
@@ -47,6 +47,9 @@ azure:
   # Secret value
   clientSecret:
 
+gcp:
+  credentials:
+  
```

Comment:
Can you add comments on what these are? Perhaps a brief explanation. I mean, is that a UUID, secret key or JSON string expected here?